src/main/java/com/exadel/discount/service/impl/DiscountServiceImpl.java

```diff
@@ -168,4 +180,20 @@ private Predicate preparePredicateForFindingAll(DiscountFilter filter) {
                         .append(filter.getVendorIds(), QDiscount.discount.vendor.id::in)
                         .buildAnd());
     }
+
+    private Predicate prepareSearchPredicate(String searchText) {
+        List<Predicate> searchPredicates = Pattern
+                .compile(" ")
+                .splitAsStream(searchText)
+                .filter(word -> word.length() >= 3)
```

Comment:
move to constants and name carefully for clear understanding what is that a magic number

Reply:
+